### PR TITLE
Add shards to genesis state

### PR DESCRIPTION
**Pull request by sgkim126**

### CODE REVIEW

**Comment by sgkim126** [comment]
@joojis This patch adds two RPC. Would you review 7f3f3dae57ec53b03b87512526167952a3bab6bb and e6b21e4b36f6cffbfc8972b5798f6e6725170594?

**Comment by sgkim126** [comment]
@kseo This patch adds shard to genesis state. Would you review c441e86407a7658752ccbc9af902a4966310fdf9 and 1769af0df1b20985f7fba4d6289d146e65892f20 ?

**Comment by sgkim126** [comment]
The commits except the above 4 commits are just structure declaration used by the 4 commits.

File: spec/JSON-RPC.md [hunk 1]
```diff
@@ -465,6 +467,50 @@ Response Example
 {"jsonrpc":"2.0","result":"0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","id":null}
 ```
 
+## chain_getNumberOfShards
+Gets the number of shards, at state of given blockNumber.
+
+Param:
+1. block number: `numbeer` or `null`
```
Comment (joojis) [inline spec/JSON-RPC.md]: number

**Pull request merged**